db/migrate/20181213214340_create_project_dependent_repositories.rb

```diff
@@ -0,0 +1,10 @@
+class CreateProjectDependentRepositories < ActiveRecord::Migration[5.2]
+  def change
+    create_view :project_dependent_repositories, materialized: true
+    add_index :project_dependent_repositories, [:project_id, :rank, :stargazers_count],
```

Comment:
Otherwise these indexes are going to have to then also iterate over the table to be created rather than getting filled in as the data refreshed the first time (concurrently)